lib/uwb/include/uwb/UwbSession.hxx

```diff
@@ -136,12 +136,21 @@ public:
     SetSessionStatus(const uwb::protocol::fira::UwbSessionStatus& status);
 
     /**
-     * @brief Get the application configuration parameters for this session.
+     * @brief Get the Application Configuration Parameters object
```

Comment:
I think the old text should be restored here. This new text looks like the canned doxygen text for a new block.